src/components/IntroductionDialog/IntroductionDialog.vue

```diff
@@ -1,50 +1,43 @@
 <template>
-  <div class="text-center">
-    <v-dialog
-      :fullscreen="$vuetify.breakpoint.mobile"
-      v-model="dialog"
-      width="500"
-    >
-      <template v-slot:activator="{ on, attrs }">
-        <v-btn fab small outlined v-bind="attrs" v-on="on">
-          <v-icon> mdi-information-symbol </v-icon>
-        </v-btn>
-      </template>
-
-      <v-card>
-        <v-card-title class="text-h5"> Introduction </v-card-title>
-
-        <v-card-text>
-          <p>
-            The IDPMR portal is the viewport of the Integrated Data Management
-            Platform for Mangrove Rehabilitation and is in use to better
-            coordinate and monitor progress of mangrove rehabilitation.
-          </p>
-          <p>
-            World Bank are working with the Coordinating Ministry of Maritime
-            Affairs and Investment (CMMAI) to enable a data-driven approach to
-            manage and monitore the rehabilitation of Indonesia’s mangrove
-            resources.
-          </p>
-          <p>
-            It comprises four main components, namely, Institutional
-            Arrangements, People, Data, and Systems. The IDPMR integrates
-            spatial and non-spatial data from various different agencies.
-          </p>
-          <p>Worldbank and CMMAI</p>
-        </v-card-text>
-
-        <v-card-actions>
-          <v-spacer></v-spacer>
-          <v-btn color="primary" @click="dialog = false">Get started</v-btn>
-        </v-card-actions>
-      </v-card>
-    </v-dialog>
-  </div>
+  <app-dialog
+    :open="dialog"
+    title="Introduction"
+    icon="mdi-information-symbol"
+    :outline-icon="true"
+    @update-open="updateDialog"
```

Comment:
Do you have a better way to do this? I need the `dialog` here to be in synced with the other one

Reply:
This could work: https://jithilmt.medium.com/vue-js-2-two-way-data-binding-in-parent-and-child-components-1cd271c501ba